course/assets/js/responsive_tables.js

```diff
@@ -3,6 +3,9 @@ function initResponsiveTables() {
   const tables = document
     .getElementById("main-content")
     .getElementsByTagName("table")
+
+  const observer = new ResizeObserver(() => calculateOverlap())
```

Comment:
`ResizeObserver` is a decent solution to this problem. The only issue with it is that while it has pretty wide support in modern browsers, there are older versions that it is not compatible with. It is also not compatible with any version of Internet Explorer. Because OCW is used in classrooms all over the world, it's important that major functionality be as backwards compatible as possible with old browsers and as compatible as possible with as many different browsers as possible, including IE. A lot of schools have computers that are locked down with older versions of Windows and IE is the only browser they would have access to. The overlap issue doesn't make the site unusable, so I think it's okay to merge this, but for critical functionality we should be as careful as possible to preserve this compatibility. Before you use any functionality like this, it's a good idea to check caniuse: https://caniuse.com/resizeobserver

Reply:
To preserve the compatibility with IE we can use [polyfill implementation of resizeObserver ](https://www.npmjs.com/package/resize-observer-polyfill)